src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -54,6 +54,18 @@ export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
       },
     };
   }
+  uischema.elements = removeUIElementWithName(
+    passwordFieldName,
+    uischema.elements,
+  );
+
+  const passwordWithConfirmationElement: PasswordWithConfirmationElement = {
+    type: 'PasswordWithConfirmation',
+    label: loc('oie.password.confirmPasswordLabel', 'login'),
+    options: {
+      input: passwordElement,
```

Comment:
how about following the `inputMeta` pattern for FieldElement?

Reply:
I made this update, but, wouldnt it be confusing to someone that looks at `inputMeta` in the `FieldElement` and see its signature, then come here and expect that same signature but to realize it is actually a `FieldElement`?

Reply:
this one might be a special case,  maybe we can add `password` and `confirmPassword` to contain uischemas for both fields, or you can derive internal uischema states. Not sure which one works better